namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-user-preferences-dev/resources/rds.tf

```diff
@@ -0,0 +1,42 @@
+variable "cluster_name" {
+}
+
+variable "cluster_state_bucket" {
+}
+
+module "hmpps_user_preferences_rds" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.12"
+  cluster_name           = var.cluster_name
+  cluster_state_bucket   = var.cluster_state_bucket
+  team_name              = var.team_name
+  business-unit          = var.business_unit
+  application            = var.application
+  is-production          = var.is_production
+  namespace              = var.namespace
+  db_engine_version      = var.db_engine_version
+  environment-name       = var.environment
+  infrastructure-support = var.infrastructure_support
+  rds_family             = var.rds_family
+
+  providers = {
+    aws = aws.london
+  }
+}
+
+resource "kubernetes_secret" "hmpps_user_preferences_rds" {
+  metadata {
+    name      = "hmpps_user_preferences-rds-instance-output"
```

Comment:
@katie-cawthorne , the kubernetes_secret name can only take  alphanumeric, - and .(dot).

Reply:
Copy/paste error, I'll fix